### DISCUSSION TITLE
Live streaming shoutcast/icecast Not getting song name from encoder

### DISCUSSION THREAD

**Discussion by caf2022** [savonet/liquidsoap#2473] [General]
Hi, I'm having a problem with some broadcast softwares like Opticodec-PC-SE/LE that is not capturing the metadata sent.

```
dj = amplify(override="replay_gain", 1.,input.harbor(id="aovivo", icy=true,"/",port=3000,auth=dj_auth, replay_metadata=true, icy_metadata_charset = "UTF-8", metadata_charset = "UTF-8", on_connect=fun (_) -> dj_connected("on",8000), on_disconnect=fun () -> dj_disconnected("off",8000), buffer = 5.00, max = 10.00))
dj = metadata.map(current_song, dj)
```
THe current song keep empty

**Comment by caf2022** [comment]
But, if I change on Opticodec from shoutcast to icecast the metadata will sent. Maybe is because Opticodec is very old and not have a option to select shoutcast v2, but with this input.harbor that I configured, the shoutcast v1 and v2 are supported right?

**Reply by toots** [reply]
Shoutcast v1 is definitely supported. I don't think that v2 is.

**Reply by caf2022** [reply]
Ok, I will continue to do some tests and update soon with the results. Thanks for the help.

If I'm not wasting your time, you could look at another discussion I created a while ago, maybe you know how to solve it and I apologize for asking this:

https://github.com/savonet/liquidsoap/discussions/2439

**Reply by toots** [reply]
You're never wasting my time, I just try to organize and prioritize according to perceived severity and available time.. 🙂
